CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/recipes/MCRecipeManager.java

```diff
@@ -806,7 +837,7 @@ public String calculateName() {
         
         @Override
         public void apply() {
-            ForgeRegistries.RECIPES.register(recipe.setRegistryName(new ResourceLocation("crafttweaker", name)));
+            ForgeRegistries.RECIPES.register(recipe.setRegistryName(new ResourceLocation(group, name)));
```

Comment:
The group should not be used as the recipe modid. Making this change will also cause Forge to spam console with errors for all of this.

Reply:
Outside of directly modifying whichever in-game guide uses a recipe reference for, this is the only simple way to allow people to edit recipes found in those guides without also breaking them. Some guide systems allow editing through ModTweaker, but others are hard-coded values. 

While it goes against the Forge convention and will indeed cause console warnings, I think the trade-off is worth it.